src/facade/conn_context.h

```diff
@@ -50,6 +52,22 @@ class ConnectionContext {
     return res;
   }
 
+  void SendError(std::string_view str, std::string_view type = std::string_view{}) {
+    rbuilder_->SendError(str, type);
+  }
+
+  void SendError(ErrorReply&& error) {
+    rbuilder_->SendError(std::move(error));
+  }
+
+  void SendError(OpStatus status) {
+    rbuilder_->SendError(status);
+  }
+
+  void SendSimpleString(std::string_view str) {
+    rbuilder_->SendSimpleString(str);
+  }
```

Comment:
One day I'll make ErrorReply's constructor implicit and remove all the overloads 😆

Reply:
it could be nice.